src/java.base/share/classes/java/util/Collections.java

```diff
@@ -442,8 +461,8 @@ public static void shuffle(List<?> list) {
      * portion of the list that runs from the first element to the current
      * position, inclusive.<p>
      *
-     * This method runs in linear time.  If the specified list does not
-     * implement the {@link RandomAccess} interface and is large, this
+     * @implSpec This method runs in linear time.  If the specified list does 
```

Comment:
Please also convert the corresponding paragraph in the
shuffle(List<?> list)
method to be an implSpec tag.

Reply:
@jddarcy sure. Should I mention this in CSR?

Reply:
Ah, I see that you already updated it. Thanks!